mirrord/layer/Cargo.toml

```diff
@@ -41,7 +41,8 @@ tokio-util.workspace = true
 thiserror.workspace = true
 trust-dns-resolver.workspace = true
 rand = "0.8"
-fancy-regex = { version = "0.10" }
+fancy-regex = "0.10"
+regex = "1"
```

Comment:
`regex` doesn't support some useful regex operations. One that comes to mind and triggered me changing this was "Skip /etc/*, except if it's /etc/foo.txt".

Reply:
But if we're ok with losing functionality, then `regex` is fine.

Reply:
doesn't `|` cover it?

Reply:
nvm

Reply:
I went with the regex because of regexset making way more sense.

Reply:
The easiest solution to the example I gave involves using "lookaheads", which `regex` doesn't support.